src/components/Preview.jsx

```diff
@@ -28,16 +28,20 @@ export default function Preview({
 
   return (
     <div className="preview output__item">
-      <div className="preview__title-bar">
-        <span
-          className="preview__button preview__button_pop-out u__icon"
-          onClick={onPopOutProject}
-        >&#xf08e;</span>
-        {title}
-        <span
-          className="preview__button preview__button_reset u__icon"
-          onClick={onRefreshClick}
-        >&#xf021;</span>
+      <div className="preview__title-bar sub-bar">
+        <div className="sub-bar__left">
+          {title}
+        </div>
+        <div className="sub-bar__right">
+          <span
+            className="sub-bar__button"
+            onClick={onPopOutProject}
+          >Full screen</span>
```

Comment:
I find the icons more usable here (the eye finds what it’s looking for more quickly if it doesn’t have to read the words). And I have never noticed students finding them unintuitive. Have you?

Reply:
Just looking for ways to increase overall clarity and discoverability. Anecdotally I see a lot of students don't know that those icons are even actions at all, let alone what they do. Would you be averse to icon + text?